style.css

```diff
@@ -116,3 +116,348 @@
+}
+
+.project-description > .Project1-text2 {
+  position: static;
+  width: 99px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-description > .Project1-text3 {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-requirement {
+  font: normal normal 400 15px 'Poppins', sans-serif;
+  line-height: 24px;
+  color: #344563;
+  position: static;
+  width: 295px;
+  height: 76px;
+  left: 0;
+  top: 92px;
+}
+
+.Project1-languages {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 10px;
+  list-style: none;
+  float: initial;
+}
+
+.Project1-languages > .lang1 {
+  float: left;
+  width: 53px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+.Project1-languages > .lang2 {
+  float: left;
+  width: 40px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+.Project1-languages > .lang3 {
+  float: right;
+  width: 95px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+.btn {
+  width: 124px;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  border: 2px solid #6070ff;
+  color: #6070ff;
+  background-color: #fff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin-top: 24px;
+  padding-left: 12px;
+}
+
+.btn:hover {
+  top: 12px;
+  background: #6070ff;
+  box-shadow: 0 8px 16px #172b4d;
+  color: #fff;
+}
+
+.btn:active {
+  top: 12px;
+  background: #2230d2;
+  box-shadow: 0 8px 16px #172b4d;
+  color: #fff;
+}
+
+#about-container {
+  display: grid;
+  padding: 150px 30px 150px 30px;
+  background-image: url("");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top right;
+}
+
+.about-content {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: auto;
+}
+
+.title {
+  position: static;
+  width: 327px;
+  height: 101px;
+  left: 0;
+  top: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.about-description {
+  font: normal normal 400 16px 'Poppins', sans-serif;
+  position: static;
+  width: 370px;
+  height: 144px;
+  left: 0;
+  top: 113px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
```

Comment:
- [ ] You have a horizontal scrollbar. Please fix the styling here as in the image. :+1:
![horizontal-scroll-fixed](https://user-images.githubusercontent.com/93039359/176936902-610de062-3f3b-4693-a690-574f2942307a.png)

Reply:
@HAMMAS-SALEEM Ok got it! Thank you🙏